projects/drift.js

```diff
@@ -16,7 +16,7 @@ async function tvl(_, _b, _cb, { api, }) {
 
   const legacyVaults = [
     '6W9yiHDCW9EpropkFV8R3rPiL8LVWUHSiys3YeW6AT6S', // legacy usdc vault
-    '2CqkQvYxp9Mq4PqLvAQ1eryYxebUh4Liyn5YMDtXsYci', // legacy usdc insurance fund
+    'Bzjkrm1bFwVXUaV9HTnwxFrPtNso7dnwPQamhqSxtuhZP', // legacy usdc insurance fund
```

Comment:
this looks like invalid address.

I copied your change from here: https://github.com/DefiLlama/DefiLlama-Adapters/pull/6909/files#diff-0ea3688383fd8f783666f4ad6ce9697f539e9a6c75b9e40feeb2a35ff4c7fd4dR44

Reply:
Omg sorry someone else in my org asked me to make that change and didn't double check it .. will figure out what the problem is

Reply:
Fixed .. I accidentally added a 'P' on the end 😆 , appreciate your help :)